src/main/java/org/icgc/argo/workflow_management/service/NextflowService.java

```diff
@@ -93,8 +93,11 @@ private String startRun(WESRunParams params)
   private String cancelRun(@NonNull String runId) {
     val masterUrl = config.getK8s().getMasterUrl();
     val namespace = config.getK8s().getNamespace();
-    val config = new ConfigBuilder().withMasterUrl(masterUrl).withNamespace(namespace).build();
-
+    val config = new ConfigBuilder()
+                .withTrustCerts(true)
```

Comment:
make this boolean configurable

Reply:
I'm not sure  if `withTrusstCerts` set to true will work yet, will make this configurable once I test the fix in fartingowl.

Reply:
:+1: